tests/test_remove_python2_unicode_compatible.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests for verifying the output of the script edx_repo_tools/codemods/django3/remove_python2_unicode_compatible.py
+"""
+import os
+from unittest import TestCase
+import shutil
+import uuid
+import subprocess
+
+
+class Python2UnicodeCompatibleTests(TestCase):
+    """
+    Tests for the Python2_unicode_compatible.py output
+    """
+    def setup_local_copy(self, file_name):
+        """
+        Setup local copy of the sample file for tests
+        """
+        current_directory = os.path.dirname(__file__)
+        temp_file = os.path.join(current_directory, 'sample_files', str(uuid.uuid4()) + ".py")
+        local_file = os.path.join(current_directory,  'sample_files', file_name)
+        shutil.copy2(local_file, temp_file)
+        return temp_file
+
+    def test_python2_unicode_compatible_removed(self):
+        """
+        Test the script on a sample file to make sure it is removing python_2_unicode_compatible imports and decorators.
+        """
+        sample_file_path = self.setup_local_copy("sample_python2_unicode_removal.py")
+        with open(sample_file_path) as sample_file:
+            sample_code = sample_file.read()
+            assert "python_2_unicode_compatible" in sample_code
+        subprocess.call(['remove_python2_unicode_compatible', sample_file_path])
```

Comment:
why don't we call the modernizer script directly here

Reply:
Updated the function implementation according to the suggestion.